app/locale/fr.coffee

```diff
@@ -1764,13 +1767,13 @@ module.exports = nativeDescription: "français", englishDescription: "French", t
 #    subheader: "You can make your licenses available to other teachers in your organization. Each license can only be used for one student at a time."
 #    teacher_not_found: "Teacher not found. Please make sure this teacher has already created a Teacher Account."
 #    teacher_not_valid: "This is not a valid Teacher Account. Only teacher accounts can share licenses."
-    already_shared: "Vous avez déjà partagé ces licences avec ce Professeur."
+#    already_shared: "You've already shared these licenses with that teacher."
```

Comment:
Example of translation we would lose because submitter missed uncommenting its section header.